extractor/src/main/java/org/schabi/newpipe/extractor/utils/LocaleCompat.java

```diff
@@ -0,0 +1,36 @@
+package org.schabi.newpipe.extractor.utils;
+
+import java.util.Locale;
+
+public class LocaleCompat {
+    private LocaleCompat() {
+    }
+
+    // Source: LocaleListCompat's private forLanguageTagCompat() method.
+    // Use Locale.forLanguageTag() on API level >= 21 instead.
```

Comment:
Precise the comment to make sure that we understand this is the Android API level, not the Java one.